### PR TITLE
EthanA_changes

### PR DESCRIPTION
**Pull request by AnsaleE**
I am still unsure what this merge thing is, but it really isn't going away

### CODE REVIEW

**Comment by zhavens** [comment]
Nothing to worry about with the merges. This is just what it looks like when you're merging other people's changes that have been pulled into `main` into your development branch (usually by doing a rebase or a fetch). 

If you want more details, this person has answered the question well: https://stackoverflow.com/questions/31463025/why-does-git-merge-a-branch-into-itself

File: source/Sequences/DisabmiguatingSquareBrackets.ptx [hunk 1]
```diff
@@ -22,7 +22,7 @@ part_of_new_lst = new_lst[0]
...
+  <p>In the code above, both lines use square brackets, but one uses them for list creation and one uses them for indexing. You can differentiate them because with list creation there is nothing between the equal sign and the opening square bracket. On the left side of the equal sign are the variable that the list is being assigned to, and on the right side of the equal sign is the two square brackets with items in the list in between them. On the second line, the square brackets come immediately after a variable name, which tells you that they are indexing into that list variable. Because we have elements inside of <c>new_lst</c>, we can index into it.
```
Comment (celinelatulipe) [inline source/Sequences/DisabmiguatingSquareBrackets.ptx:25]: No, this is not correct. It should be 'is' not 'are'

File: source/Iteration/WPKeepingTrackofYourIteratorVariableYourIterable.ptx [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <section xml:id="iteration_keeping-track-of-your-iterator-variable-and-your-iterable">
   <title>&#x1F469;&#x200D;&#x1F4BB; Keeping Track of Your Iterator Variable and Your Iterable</title>
   <p>When students first begin using for loops, they sometimes have difficulty understanding the difference between the iterator variable (the loop variable) and the iterable.</p>
-  <p>The iterable is the object that you will parsing through in a for loop, typically a String or list, or other type of sequence. Generally, this object does not change while the for loop is being executed.</p>
+  <p>The iterable is the object that you will be parsing through in a for loop, typically a string, list, or other type of sequence. Generally, this object does not change while the for loop is being executed.</p>
```
Comment (celinelatulipe) [inline source/Iteration/WPKeepingTrackofYourIteratorVariableYourIterable.ptx:5]: We want to use String here, since String is a type.

**Review by celinelatulipe — CHANGES_REQUESTED** [review]
There are two issues in these changes that don't work and need to be addressed before I can merge. Hopefully you can see those comments.

**Review by celinelatulipe — APPROVED** [review]

**Pull request merged**